services/movie_session.py

```diff
@@ -0,0 +1,54 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession, CinemaHall, Movie
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+    movie = Movie.objects.get(id=movie_id)
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie=movie,
+        cinema_hall=cinema_hall,
+    )
```

Comment:
```suggestion
    MovieSession.objects.create(
        show_time=movie_show_time,
        movie_id=movie_id,
        cinema_hall_id=cinema_hall_id
    )
```
you can do without additional querys

Reply:
But I can't just use movie_id and cinema_id because the model expects an instance of CinemaHall and Movie
class MovieSession(models.Model):
    show_time = models.DateTimeField()
    cinema_hall = models.ForeignKey(CinemaHall, on_delete=models.CASCADE)
    movie = models.ForeignKey(Movie, on_delete=models.CASCADE)